setup.py

```diff
@@ -82,8 +82,8 @@ def _core_version(plugin_version: str = _plugin_version()) -> str:
     packages=find_namespace_packages(include=["dbt", "dbt.*"]),
     include_package_data=True,
     install_requires=[
-        f"dbt-core~={_core_version()}",
-        f"dbt-postgres~={_core_version()}",
+        "dbt-core~=1.5.0a1",
```

Comment:
is this intentional?

Reply:
It is, and we don't intend to merge this PR as-is. All of the changes to dev-requirements.txt will be reverted before merge.

The problem is that the changeset in dbt-core which this PR coordinates with is branched from dbt-core 1.5.0a1, and I don't know if there is a better way to get the dependencies to work out so that tests pass in CI while cross-repo work is in progress. I don't know what final merge strategy is going to be, but as lead on this, @MichelleArk could provide more detail.

Reply:
Final merge strategy will be to merge the [core changes](https://github.com/dbt-labs/dbt-core/pull/6986) first, revert any changes made in adapter branches to for testing (e.g. dev-requirements.txt, setup.py changes), run CI against the latest main branch of core on adapter PRs, and merge on green.